src/content/blog/astro-590.mdx

```diff
@@ -0,0 +1,228 @@
+        // Assume each entry has a 'content' field with Markdown content
+        store.set(entry.id, {
+          id: entry.id,
+          data: entry,
+          rendered: await renderMarkdown(entry.content),
+        });
+      }
+    },
+  };
+}
+```
+
+Now, even with your custom loader, you can access `render()` and `<Content />` just as if your Markdown content were stored locally in your project:
+
+```astro
+---
+import { getEntry, render } from 'astro:content';
+const entry = await getEntry('my-collection', Astro.params.id);
+const { Content } = await render(entry);
+---
+<Content />
+```
+
+For more details, see the [content loaders documentation](https://docs.astro.build/en/reference/content-loader-reference/#rendermarkdown).
+
+## Disable default styles in experimental responsive images
+
+When using [experimental responsive images](https://docs.astro.build/en/reference/experimental-flags/responsive-images/), Astro applies default styles to ensure the images resize correctly. In most cases this is what you want – and they are applied with low specificity so your own styles override them.
+
+However in some cases you may want to disable these default styles entirely. This is particularly useful when using Tailwind 4, because it uses CSS cascade layers to apply styles, making it difficult to override Astro's default styles.
+
+Astro 5.8.1 added a new `image.experimentalDefaultStyles` boolean configuration option for applying these default styles. It defaults to `true`, providing the current responsive image behavior. But if you'd rather not fight with Tailwind 4, setting `!important` styles everywhere, you can simply disable this option in your Astro config:
+
+```js
+export default {
+  image: {
+    experimentalDefaultStyles: false,
+  },
+  experimental: {
+    responsiveImages: true,
+  },
+};
+```
+
+For more details, see the [experimental responsive images documentation](https://docs.astro.build/en/reference/experimental-flags/responsive-images/#overriding-image-styles).
+
+## Allow adapters to suppress logs about feature support
+
+[Astro adapters](https://docs.astro.build/en/reference/adapter-reference/) can [declare whether or not they support certain Astro features](https://docs.astro.build/en/reference/adapter-reference/#building-an-adapter), and whether that support is stable or experimental. 
+
+During builds, Astro will log a warning or error if a site is using features that are not supported by the adapter. However, sometimes an adapter may need to log more specific messages to help users resolve the issue. Previously this could be confusing, as these custom logs were printed in addition to the Astro-generated ones and may appear to contradict them. Astro 5.9 adds an option to allow adapters to suppress the logging for unsupported features.
+
+To do so, an adapter author can add `suppress: "all"` (to suppress both the default and custom message) or `suppress: "default"` (to only suppress the default one):
```

Comment:
```suggestion
During builds, Astro will log a warning or error if a site is using features that are not supported by the adapter. However, sometimes an adapter may need to log more specific messages to help users resolve the issue. Previously this could be confusing, as these custom logs were printed in addition to the Astro-generated ones and may appear to contradict them.

Astro 5.9 adds an option to allow adapters to suppress the logging for unsupported features. An adapter author can add `suppress: "all"` (to suppress both the default and custom message) or `suppress: "default"` (to only suppress Astro's default message):
```
Whenever we can, starting a paragraph with "Astro x.x" is good for scanning and getting to the new thing!